scripts/fetch/fetch.nws_conversion.R

```diff
@@ -0,0 +1,20 @@
+fetch.nws_conversion <- function(viz = as.viz("nws-conversion")){
+  states <- viz[['states']]
+  
+  conversion.table.all <- data.frame()
+  for(s in states) {
+    conversion.url <- sprintf("http://www.nws.noaa.gov/oh/hads/USGS/%s_USGS-HADS_SITES.txt", s)
+    conversion.table.state <- readr::read_delim(conversion.url,
+                                                delim = "|",skip = 4,col_names = FALSE)
+    conversion.table.all <- dplyr::bind_rows(conversion.table.all, conversion.table.state)
+  }
+  
+  
+  names(conversion.table.all) <- c("NWS","USGS","GOES","NWS HSA","lat","lon","name")
+  conversion.table.all$USGS <- gsub(" ","", conversion.table.all$USGS)
+  
+  location <- viz[['location']]
+  saveRDS(conversion.table.all, file=location)
+}
+
+fetchTimestamp.nws_conversion <- vizlab:::fetchTimestamp.file
```

Comment:
i think the right option for this will ultimately be `fetchTimestamp.url` or a wrapper of that method, but i'd like to think about it for a while first. this is good for now: should only write the file once to a person's computer unless this file or other dependencies of 'nws-conversion' change.

Reply:
yeah, and this is a case where this particular data is not terribly dynamic.

Reply:
sounds good